tests/test_main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_cat_age_converted_to_human_age_correctly():
```

Comment:
Please create a test for each case (class of equivalence)

Reply:
Did you mean that I should write separate test function for each assertion? Is it really necessary?